src/js/components/Popup/PopupTooltip.tsx

```diff
@@ -30,14 +30,26 @@ const PopupTooltip = ({
   const buttonRef = React.useRef<HTMLButtonElement | null>(null);
   const tooltipRef = React.useRef<HTMLDivElement | null>(null);
   const [isPopupOpen, setIsPopupOpen] = React.useState(false);
-  const [center, setCenter] = React.useState<{ centerX: number, centerY: number }>({ centerX: 0, centerY: 0 });
+  const [center, setCenter] = React.useState<{ centerX: number; centerY: number }>({
+    centerX: 0,
+    centerY: 0,
+  });
   const describedby = ariaDescribeBy ? ariaDescribeBy.toLocaleLowerCase().replace(/ /g, '-') : null;
 
   const centerContent = React.useCallback(() => {
     if (buttonRef && buttonRef.current) {
-      const { left, top, width, height } = buttonRef.current.getBoundingClientRect()
-      const centerX = left + width / 2;
-      const centerY = top - height;
+      const { left, top, width, height } = buttonRef.current.getBoundingClientRect();
+      const closestScrollableParent = buttonRef.current.closest('.Modal-body'); // Assuming the modal (or any scrollable container) has a class of `.scrollable`
+      console.log('closestScrollableParent', closestScrollableParent);
```

Comment:
Hey @jondauz I know this was already merged but in a future PR please remove this console. If I make a change in teamsnap-ui soon, I can remove my self